news/10709.process.rst

```diff
@@ -1,3 +1,4 @@
 Start migration of distribution metadata implementation from ``pkg_resources``
-to ``importlib.metadata``. The new implementation is currently not exposed in
-any user-facing way, but included in the code base for easier development.
+to ``importlib.metadata``. The new implementation is enabled by default on
+Python 3.11 (or later), but can be disabled with a private environment variable
+``_PIP_USE_IMPORTLIB_METADATA``.
```

Comment:
If we are documenting it, it's not a private variable. I'd rather we make the variable public in that case.